app/src/main/java/nl/tudelft/b_b_w/view/ContactAdapter.java

```diff
@@ -169,7 +169,7 @@ public View getView(final int position, View convertView, ViewGroup parent) {
             ibanItemText.setText(blockController.getBlocks(ownerName).get(position).getOwner().getIban());
             ImageView pic = (ImageView) view.findViewById(R.id.trust_image);
             pic.setImageResource(
-                    getImageNo(blockController.getBlocks(ownerName).get(position).getTrustValue()));
+                    getImageNo((int) Math.ceil(blockController.getBlocks(ownerName).get(position).getTrustValue())));
```

Comment:
I think Math.ceil isn't doing what you want. Math.ceil always rounds the number upwards, I guess you want to use Math.round.